testnet/stop-obscuro-node.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+#
+# This script gracefully stops the Obscuro node.
+#
+
+help_and_exit() {
+    echo ""
+    echo "Usage: "
+    echo "      -  $(basename "${0}") "
+    echo ""
+    exit 1  # Exit with error explicitly
+}
+# Ensure any fail is loud and explicit
+set -euo pipefail
+
+# Define local usage vars
+start_path="$(cd "$(dirname "${0}")" && pwd)"
+testnet_path="${start_path}"
+
+# Fetch options
+for argument in "$@"
+do
+    key=$(echo $argument | cut -f1 -d=)
+    value=$(echo $argument | cut -f2 -d=)
+
+    case "$key" in
+
+            --help)                     help_and_exit ;;
+            *)
+    esac
+done
+
+
+docker-compose stop
```

Comment:
Does this stop all running containers gracefully? i.e. what would happen if the user was running other containers on the same host?

Reply:
So you have to be in the testnet folder, and then it will automatically resolve to using the docker-compose.yml file?

Reply:
Running locally I don't think that then removes the old containers does it, so restarting would require you to do a docker container prune?

Reply:
Yes, that's my understanding.